ui/src/app/views/notebook-list/component.ts

```diff
@@ -45,7 +45,7 @@ export class NotebookListComponent implements OnInit, OnDestroy {
   showTip: boolean;
   cohortsLoading: boolean;
   cohortsError: boolean;
-  newPageVisit: PageVisit;
+  newPageVisit: PageVisit = { page: NotebookListComponent.PAGE_ID };
```

Comment:
Sorry, I was not entirely clear with my comment. I meant that I would remove the `newPageVisit` variable entirely and just inline this object at the callsite below. It's not needed anywhere else.

(here and in the other file)